src/Bindings/ManualBindings.cpp

```diff
@@ -2895,7 +2887,74 @@ class cLuaBlockTracerCallbacks :
 
 protected:
 	cLuaState::cTableRefPtr m_Callbacks;
-} ;
+};
+
+
+
+
+
+/** Provides interface between a Lua table of callbacks and the cBlockTracer::cCallbacks */
+class cLuaBlockTracerCallbacksOld :
+	public cLuaBlockTracerCallbacks
+{
+public:
+	cLuaBlockTracerCallbacksOld(cLuaState::cTableRefPtr && a_Callbacks):
+		cLuaBlockTracerCallbacks(std::move(a_Callbacks))
+	{
+	}
+
+	virtual bool OnNextBlock(Vector3i a_Block, BLOCKTYPE a_BlockType, NIBBLETYPE a_BlockMeta, eBlockFace a_EntryFace) override
+	{
+		bool res = false;
+		if (
+			m_Callbacks->CallTableFn("OnNextBlock", a_Block.x, a_Block.y, a_Block.z, a_BlockType, a_BlockMeta, a_EntryFace, cLuaState::Return, res)
+		)
```

Comment:
Now this definitely deserves breaking into multiple lines.

Reply:
Done